scripts/train2/inference/detector.py

```diff
@@ -748,7 +746,7 @@ def find_objects(vertex2, aff, config, numvertex=8, run_sampling=False, num_samp
                             best_angle = dist_angle
                             best_dist = dist_point
 
-                    if i_best is -1:
```

Comment:
you dont like `is`?

Reply:
Python doesn't like it in this case. Python 3.8.10 threw a SyntaxWarning (something like `'is' used with literal`).